UnityProject/Assets/Scripts/Learning/ProtipObject.cs

```diff
@@ -30,18 +30,22 @@ private void Awake()
 			if(saved == "true" || TipSO == null) Destroy(this);
 		}
 
-		protected virtual bool TriggerConditions()
+		protected virtual bool TriggerConditions(GameObject triggeredBy)
 		{
+			//To avoid issues with NREs, Protips should only trigger if a PlayerScript exists.
+			if (PlayerManager.LocalPlayerScript == null) return false; 
```

Comment:
AIs have local scripts, but do ghosts not have it? Either way ghosts shouldn't receive tips by design, only when the player is still dead in his body.

Reply:
why not? Maybe we do want to tell them stuff like how the ghost roles work or interactions they might be able to do in the future.

Reply:
we can probably make an protip object specfic to ghosts then since protips currently rely on the existence of a local script and the game likes to complain if it cant find it.